assets/xml/objects/object_f40_obj.xml

```diff
@@ -1,17 +1,25 @@
 ﻿<Root>
+    <!-- 
+        Assets for Stone Tower platforms. These platforms include the cubes moved by pressing the switches in 
+        Stone Tower and two unused platforms.
+    -->
     <File Name="object_f40_obj" Segment="6">
-        <Texture Name="object_f40_obj_Tex_000000" OutName="tex_000000" Format="i8" Width="64" Height="64" Offset="0x0" />
-        <Texture Name="object_f40_obj_Tex_001000" OutName="tex_001000" Format="rgba16" Width="32" Height="64" Offset="0x1000" />
-        <Texture Name="object_f40_obj_Tex_002000" OutName="tex_002000" Format="rgba16" Width="32" Height="32" Offset="0x2000" />
-        <Texture Name="object_f40_obj_Tex_002800" OutName="tex_002800" Format="i4" Width="32" Height="128" Offset="0x2800" />
-        <Texture Name="object_f40_obj_Tex_003000" OutName="tex_003000" Format="i4" Width="64" Height="64" Offset="0x3000" />
-        <DList Name="object_f40_obj_DL_003B00" Offset="0x3B00" />
-        <DList Name="object_f40_obj_DL_003B08" Offset="0x3B08" />
-        <Collision Name="object_f40_obj_Colheader_003E80" Offset="0x3E80" />
-        <DList Name="object_f40_obj_DL_004030" Offset="0x4030" />
-        <DList Name="object_f40_obj_DL_004038" Offset="0x4038" />
-        <Collision Name="object_f40_obj_Colheader_004240" Offset="0x4240" />
-        <DList Name="object_f40_obj_DL_0043D0" Offset="0x43D0" /> <!-- Original name is "z2_f40obj01_model" -->
-        <Collision Name="object_f40_obj_Colheader_004640" Offset="0x4640" /> <!-- Original name is "z2_f40obj01_bgdatainfo" -->
+        <Texture Name="gStoneTowerPlatformsTopTex" OutName="stone_tower_platforms_top" Format="i8" Width="64" Height="64" Offset="0x0" /> <!-- Unused -->
+        <Texture Name="gStoneTowerPlatformsSidesTex" OutName="stone_tower_platforms_sides" Format="rgba16" Width="32" Height="64" Offset="0x1000" /> <!-- Unused -->
+        
+        <Texture Name="gStoneTowerBlockBackgroundTex" OutName="stone_tower_block_background" Format="rgba16" Width="32" Height="32" Offset="0x2000" />
+        <Texture Name="gStoneTowerBlockPatternTex" OutName="stone_tower_block_pattern" Format="i4" Width="32" Height="128" Offset="0x2800" />
+        <Texture Name="gStoneTowerBlockSidesTex" OutName="stone_tower_block_sides" Format="i4" Width="64" Height="64" Offset="0x3000" />
+        
+        <DList Name="gStoneTowerEmpty1DL" Offset="0x3B00" />
+        <DList Name="gStoneTowerVerticallyOscillatingPlatformDL" Offset="0x3B08" /> <!-- Unused -->
+        <Collision Name="gStoneTowerVerticallyOscillatingPlatformCol" Offset="0x3E80" /> <!-- Unused -->
```

Comment:
```suggestion
        <DList Name="gStoneTowerVerticallyOscillatingPlatformEmptyDL" Offset="0x3B00" />
        <DList Name="gStoneTowerVerticallyOscillatingPlatformDL" Offset="0x3B08" /> <!-- Unused -->
        <Collision Name="gStoneTowerVerticallyOscillatingPlatformCol" Offset="0x3E80" /> <!-- Unused -->
```
Because there are many actors for `StoneTower`, I think I'd prefer this? Empty DLists can be used for opa/xlu that aren't used, so it still makes sense